### ISSUE TITLE
NativeImportExportPlugin over web works, but over CLI fatal: Invalid XML

### ISSUE THREAD

**Issue by ErikI1** [pkp/pkp-lib#8054]
**Describe the bug**
export over NativeImportExportPlugin is working, but not over cli (from 9 exports of issue only 2 was successful) :

 php ./tools/importExport.php  NativeImportExportPlugin export /tmp/test.xml phat issue 31

`Validation errors:
Element '{http://pkp.sfu.ca}issues': Missing child element(s). Expected is ( {http://pkp.sfu.ca}issue ).
Invalid XML:
<?xml version="1.0"?>
<issues xmlns="http://pkp.sfu.ca" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pkp.sfu.ca native.xsd"/>

PHP Fatal error:  Uncaught Exception: Could not convert selected objects. in /opt/ojs/ojs-3.3.0-11/lib/pkp/classes/plugins/ImportExportPlugin.inc.php:181
Stack trace:
#0 /opt/ojs/ojs-3.3.0-11/plugins/importexport/native/NativeImportExportPlugin.inc.php(294): ImportExportPlugin->displayXMLValidationErrors()
#1 /opt/ojs/ojs-3.3.0-11/plugins/importexport/native/NativeImportExportPlugin.inc.php(462): NativeImportExportPlugin->exportIssues()
#2 /opt/ojs/ojs-3.3.0-11/tools/importExport.php(63): NativeImportExportPlugin->executeCLI()
#3 /opt/ojs/ojs-3.3.0-11/tools/importExport.php(69): importExport->execute()
#4 {main}
  thrown in /opt/ojs/ojs-3.3.0-11/lib/pkp/classes/plugins/ImportExportPlugin.inc.php on line 181
`

**To Reproduce**
1. download export from https://drive.google.com/file/d/1Fl4IW2d7_8PmD75qbP5IOarBoNK-luCJ/view?usp=sharing
2. try import a export over cli

**What application are you using?**
ojs-3.3.0-11
php 7.4

**Comment by asmecher** [comment]
@defstat, could you take a look at this issue?

**Comment by defstat** [comment]
Hi @ErikI1, thanks for reporting that. I have two questions to start working on it:
1. Is your problem Importing the data, or exporting the data. I am asking because you write 
> export over NativeImportExportPlugin is working, but not over cli

but then in the steps to reproduce you 

> try import a export over cli

2. I tried to import the provided exported XML file to my dev installation (stable_3.3.0 branch) and it was successfully imported. 
![image](https://user-images.githubusercontent.com/560456/177938048-a2485c33-e496-4b6e-a0a7-6a30752e495f.png)

**Comment by ErikI1** [comment]
Hi, thank you for your time.
1. problem is with export over cli (it would be better if I put the command in quotation marks)
2.  www-data@e538414921e9:/opt/ojs/app$ php ./tools/upgrade.php  check
Code version:      3.3.0.11
Database version:  3.3.0.11
Latest version:    3.3.0.11
Your system is up-to-date

I apologize for my English and misunderstanding.

**Comment by ErikI1** [comment]
[https://drive.google.com/file/d/1xfu3xujyCGdw2FOzjKjc-LQ05-R7hN6H/view?usp=sharing](url)

**Comment by defstat** [comment]
Hi @ErikI1.. Thanks for the reply. I don't see any issues there either. I downloaded and installed [OJS 3.3.0-11 LTS (.tar.gz)](https://pkp.sfu.ca/ojs/download/ojs-3.3.0-11.tar.gz) locally and imported the initially given exported file. 

Then I exported the file by using the CLI command, and it was exported with no problems. So I can't reproduce the issue neither on the `stable-3.3.0` branch nor on the `OJS 3.3.0-11 release`.

Could it be that the issue you are trying to export has a different ID than the given one (not 31 in your DB)
